route-rs-runtime/src/link/mod.rs

```diff
@@ -33,7 +33,14 @@ mod blackhole_link;
 pub use self::blackhole_link::*;
 
 /// The type that all links take as input, or provide as output. This allows links to be combinatorial
-pub type PacketStream<Input> = Box<dyn Stream<Item = Input, Error = ()> + Send>;
+pub type PacketStream<Input> = Box<dyn Stream<Item = Input, Error = ()> + 'static + Send>;
```

Comment:
Added `'static` since these `Streams` should always last the lifetime of the router.

Reply:
Is this required that they be static? I am fine with it but usually static is a sign that you could have a better lifetime bound.

Reply:
We can always try to remove static later though, it doesn't seem like an issue.